### ISSUE TITLE
On the tablet, text is outside the circle

### ISSUE THREAD

**Issue by nightstorm99** [sjwall/MaterialTapTargetPrompt#32]
Hello,

On the tablet, the text is displayed outside the circle when the object is in the center of the screen.

my code:
```java
final MaterialTapTargetPrompt.Builder tapTargetPromptBuilder = new MaterialTapTargetPrompt.Builder(this)
                .setPrimaryText(R.string.ansicht_wechseln)
                .setSecondaryText(R.string.kalender_show_text2)
                .setAnimationInterpolator(new FastOutSlowInInterpolator())
                .setMaxTextWidth(R.dimen.tap_target_menu_max_width)
                .setIconDrawable(new IconicsDrawable(this, CommunityMaterial.Icon.cmd_view_module).actionBar().color(Color.WHITE))
                .setOnHidePromptListener(new MaterialTapTargetPrompt.OnHidePromptListener() {
                    @Override
                    public void onHidePrompt(MotionEvent event, boolean tappedTarget) {
                        event.setAction(MotionEvent.ACTION_CANCEL);
                    }

                    @Override
                    public void onHidePromptComplete() {
                        showTapTarget4();
                    }
                });
        tapTargetPromptBuilder.setTarget(bottomNavigation.getViewAtPosition(2));
        tapTargetPromptBuilder.show();
```

![device-2017-01-11-200325](https://cloud.githubusercontent.com/assets/10620782/21862361/2b36ae50-d839-11e6-85cd-ddf2319e6f8e.png)

**Comment by sjwall** [comment]
I can replicate this issue and will fix it so that it follows the guidelines for positioning in the center as raised in #3

**Comment by sjwall** [comment]
This issue should be fixed with the changes made in #3 and released in v1.8.0

Please let me know if it fixes it for you.

**Comment by nightstorm99** [comment]
All okay, thanks
